spec/index.html

```diff
@@ -80,7 +80,7 @@
     section. Since the edits to this document do not invalidate
     previous implementations the Director decided no new implementation report was required. </p>
 
-  <section id="related" data-include="./common//related.html"></section>
+  <section id="related" data-include="./common/related.html"></section>
```

Comment:
This does not work (for me at least) when viewed locally (all specs). I get a reSpec error and a reSpec warning.

Is there some magic I'm missing?